3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,60 @@
+import services from '../../../src/app/lib/config/services';
+import {
+  ukBasePath,
+  ukBasePathTest,
+  basePath,
+  basePathTest,
+} from '../support/env';
+
+Object.keys(services).forEach(service => {
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+
+  if (notWSServices.includes(service)) {
+    return;
+  }
+
+  // Do not run the redirect tests on the local environment
+  if (Cypress.env('APP_ENV') === 'local') {
+    return;
+  }
+
+  describe('WS Redirects', () => {
+    it(`should redirect *bbc.com/${service}`, () => {
+      let urlsTotest = [
+        `${ukBasePathTest}${service}`,
```

Comment:
You should instead be able to do this:

```
import envConfig from '../../support/config/envs';

...
      let urlsTotest = [
        `${envConfig.ukBaseUrl}${service}`,
        ...
```

Then in `envs.js` you can have:

```
{
  live: {
    ukBaseUrl: 'https://www.bbc.co.uk',
    ...
  test: {
    ukBaseUrl: 'https://www.test.bbc.co.uk',
...
  local: {
    ukBaseUrl: 'http://localhost.bbc.co.uk',
...
```

Reply:
This change is no longer required